app/bundles/PluginBundle/Bundle/PluginDatabase.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\PluginBundle\Bundle;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\ORM\Tools\SchemaTool;
+use Mautic\IntegrationsBundle\Migration\Engine;
+use Mautic\PluginBundle\Entity\Plugin;
+
+class PluginDatabase
+{
+    private readonly string $mauticDbPrefix;
+
+    public function __construct(private readonly EntityManagerInterface $em, private readonly Connection $connection, ?string $mauticDbPrefix)
```

Comment:
Do you think we could try this? https://symfony.com/blog/new-in-symfony-6-3-dependency-injection-improvements#new-options-for-autowire-attribute
```suggestion
    public function __construct(
         private readonly EntityManagerInterface $em,
         private readonly Connection $connection,
         #[Autowire(env: 'DB_TABLE_PREFIX')]
         private readonly string $mauticDbPrefix
     )
```
It's more readable than adding it to services.php

Reply:
@escopecz Thanks. Applied the suggestion with proper env variable.